docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

```diff
@@ -0,0 +1,88 @@
+
+This is typical in Rancher, as `RoleBindings` are created in the local cluster as a side effect of many operations.
+
+It is recommended to attempt reducing `RoleBindings` in the local cluster in the following ways:
+* Limit the use of the [Restricted Admin](../../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions#restricted-admin) role, preferring others wherever applicable
+* If [external authentication](../../../pages-for-subheaders/authentication-config) is configured, use groups to assign roles preferably
+* Only add users to clusters and projects when necessary
+* Remove clusters and projects when they are no longer needed
+* Only use custom roles if necessary
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider using less, but more powerful, clusters
+* Keep into account that Kubernetes permissions are always "additive" (allow-list) rather than "subtractive" (deny-list). Whenever applicable, try to minimize configurations that gives access to "all but one aspect" (cluster, project, namespace...) as that will result in the creation of a high number of `RoleBindings`
+* Experiment to see if creating new projects or creating new clusters manifests in fewer `RoleBindings` for your specific use case.
+
+### RoleBinding count estimation
+
+Predicting exactly the number of `RoleBindings` a given configuration will create depends on many factors and is complicated to calculate. However, it is possible to give a first estimation according to considerations below:
+* As a minimum estimation consider the formula `32C + U + 2UaC + 8P + 5Pa`, where `C` is the cluster count, `U` is the user count, `Ua` is the average count of users with a membership on a cluster, `P` is the project count, and `Pa` is the average number of users with a membership on a project
+* The Restricted Admin role follows a different formula, as every user with Restricted Admin role will result in at least `7C + 2P + 2` additional `RoleBindings`
+* The number of `RoleBindings` generally increases linearly with cluster count, project count, and user count
+
+### Using New Apps Over Legacy Apps
+There are two app Kubernetes resources that Rancher uses: `apps.projects.cattle.io` and `apps.cattle.cattle.io`. Legacy apps, `apps.projects.cattle.io`, were first introduced with the former UI (Cluster Manager) and are now outdated. New apps, `apps.catalog.cattle.io`, are found in the current UI (Cluster Explorer) for their respective cluster. New apps are preferable because their data resides in downstream clusters, freeing up resources in the local cluster.
```

Comment:
Just for my information: what's the current rule for sentences that begin with words that should not be capitalized?

Eg. type names (`apps.cattle.cattle.io`), project names ([etcd](https://etcd.io/docs/v3.4/faq/#what-does-the-etcd-warning-snapshotting-is-taking-more-than-x-seconds-to-finish--mean)), command names...

Reply:
My instinct was to capitalize, but checking the style guide, it's actually advised to revise so that the always-lowercase word doesn't appear at the start of a sentence